CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+Contributions
+-------------
+
+Contributions are welcome in the form of pull requests.
+
+Once the implementation of a piece of functionality is considered to be bug
+free and properly documented, it can be incorporated into the master branch.
+
+To help developing `pybv`, you will need a few adjustments to your
+installation as shown below.
+
+##### Install the development version
```

Comment:
Here don't you think we should suggest people fork and clone rather than install the organizational repo?